src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertySymbolBase.cs

```diff
@@ -1043,7 +1143,7 @@ private SynthesizedSealedPropertyAccessor MakeSynthesizedSealedAccessor()
         AttributeLocation IAttributeTargetSymbol.DefaultAttributeLocation => AttributeLocation.Property;
 
         AttributeLocation IAttributeTargetSymbol.AllowedAttributeLocations
-            => IsAutoPropertyWithGetAccessor
+            => IsAutoProperty
```

Comment:
>IsAutoProperty [](http://example.com/codeflow?start=15&length=14)

What is the motivation for this change? I couldn't find a new test or any existing test that could look related. We should stop making changes in a manner like that. Any change should come with a supported test. Compiler is a complicated machinery.

Reply:
There is an updated test that's affected by this. It's not much of an interesting scenario though, but there will certainly be more interesting scenarios for this change.

Reply:
> There is an updated test that's affected by this. It's not much of an interesting scenario though, but there will certainly be more interesting scenarios for this change.

I saw a base line change in one of the legacy tests that could be caused by this change. However, the scenario there was not related to the feature we are working on in this PR. Just affecting that scenario wouldn't be a good motivation for making this change in context of this PR. Changes that we are making should be affecting interesting scenarios in the first place and the should be tests for those interesting scenarios. We should be getting into a mode where a change is not made without a supporting test.